### PR TITLE
20463 - Feat: Support Safe List Addition and Removal

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue #:* https://github.com/bcgov/entity/issues/20463

*Description of changes:* Added a form to add email to safe list and a delete button to remove it [against each email in list].

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/views/auth/staff/SafeEmailView.vue [hunk 1]
```diff
@@ -43,13 +44,23 @@ export default defineComponent({
     }
     const safeEmails = ref<SafeEmail[]>()
 
+    async function deleteEmail (email: string) {
+      // Call the service method to delete the email from the server
+      await StaffService.deleteSafeEmail(email).then(() => {
```
Comment (seeker25) [inline auth-web/src/views/auth/staff/SafeEmailView.vue:49]: async/await pattern not work here? 

try{

}
catch {

}

File: auth-web/src/views/auth/staff/SafeEmailView.vue [hunk 2]
```diff
@@ -43,13 +44,23 @@ export default defineComponent({
     }
     const safeEmails = ref<SafeEmail[]>()
 
+    async function deleteEmail (email: string) {
+      // Call the service method to delete the email from the server
```
Comment (seeker25) [inline auth-web/src/views/auth/staff/SafeEmailView.vue:48]: redundant comment

File: auth-web/src/views/auth/staff/SafeEmailView.vue [hunk 3]
```diff
@@ -43,13 +44,23 @@ export default defineComponent({
...
       safeEmails.value = await getSafeEmails()
     })
 
     return {
-      safeEmails
+      deleteEmail,
+      safeEmails,
     }
-  }
+  },
```
Comment (seeker25) [inline auth-web/src/views/auth/staff/SafeEmailView.vue:64]: extra comma

**Review by seeker25 — APPROVED** [review]
LGTM - small linting job needs to pass as well

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2834-48hw6t8t.web.app

**Comment by seeker25** [comment]
Can't see it in preview, url is looking for dev. Squashed + merged, Well done!

**Comment by seeker25** [comment]
Notify-api is missing the delete route, probably have to implement it here:

https://github.com/bcgov/bcregistry-sre/blob/main/notify-api/src/notify_api/resources/v2/safe_list.py

**Comment by shaangill025** [comment]
> Notify-api is missing the delete route

This should be ready for review today.
https://github.com/bcgov/bcregistry-sre/pull/140

**Pull request merged**